src/lib.js

```diff
@@ -9,5 +22,29 @@ export const decodeBase16 = hexStr => {
   return Uint8Array.from(resArr)
 }
 
+/**
+ * Encode base 16 string to base 58.
+ *
+ * @param {string} hexStr
+ * @returns {string}
+ */
+export const encodeBase58 = hexStr => {
```

Comment:
A name like `lib.js` or `utils.js` is a sort of code smell.

I have been using [hex.js](https://github.com/rchain-community/liquid-democracy/blob/dev-tools/rclient/hex.js) that exports `Basic16` with `encode` and `decode` static methods. (I think that particular copy only has encode, but you get the idea).

I suppose `Base58.encode` and `.decode` could belong in the same file if it were called `codecs.js`.

Reply:
see https://github.com/rchain-community/rchain-api/blob/master/src/codec.js